commerce/domain/pricing/cart/PricingBasicCalculator/PricingCalculatorSample.cls

```diff
@@ -55,6 +56,7 @@ public class PricingCalculatorSample extends CartExtension.PricingCartCalculator
             // This is an example of throwing special type of Exception (CartCalculateRuntimeException).
             // Throwing this exception causes the rollback of all previously applied changes to the cart (in scope of given request)
             // and may not always be the best choice.
+            System.debug('Exception occurred, message:' + e.getMessage() + ', stacktrace: ' + e.getStackTraceString());
             throw new CartExtension.CartCalculateRuntimeException('An integration error occurred in COMPUTE_PRICES. Contact your admin.');
```

Comment:
We should not eat out the exception coming from Calculator. Instead you can bubble up the same exception so that it is logged with Orchestrator. Orchestrator has capability to handle it with generic message it seems.

Reply:
@ParameshOddepally: Done. Thanks for pointing this out.